ArduCopter/mode_guided.cpp

```diff
@@ -39,16 +40,19 @@ struct Guided_Limit {
     Vector3f start_pos; // start position as a distance from home in cm.  used for checking horiz_max limit
 } guided_limit;
 
-// guided_init - initialise guided controller
+// init - initialise guided controller
 bool ModeGuided::init(bool ignore_checks)
 {
     // start in position control mode
-    pos_control_start();
+    velaccel_control_start();
+    posvelaccel_update_time_ms = millis();
```

Comment:
this "posvelaccel_update_time_ms" is used to ensure that we've received a position-velocity-accel target from the user within 3 seconds.  I don't think it is necessary to intiialise it here.

Reply:
true, removed.